saleor/core/telemetry/__init__.py

```diff
@@ -23,8 +26,23 @@ def load_object(python_path: str) -> Any:
     return getattr(import_module(module), obj)
 
 
+def otel_configure_sdk():
+    configurator = _OTelSDKConfigurator()
+    configurator.configure(resource_attributes={SERVICE_INSTANCE_ID: str(uuid.uuid4())})
+
+
+TELEMETRY_DISABLE_AUTO_CONFIGURE = (
+    os.environ.get("TELEMETRY_DISABLE_AUTO_CONFIGURE", "").lower() == "true"
+)
```

Comment:
Why is this an env variable? Do we expect people to choose? Are both values supported?

Reply:
- This is an env variable to avoid potential conflicts with Django if we decide to use Django auto-instrumentation in the future
- The idea was to provide a simple mechanism for disabling OpenTelemetry's automatic configuration when needed (for example, during development). We do not expect users to use this option frequently
- Both values are supported. Auto-configuration runs by default and is disabled only when this env variable is set